GPU/GPUTracking/Standalone/display/GPUDisplayBackendGlfw.cxx

```diff
@@ -32,8 +32,13 @@ extern "C" int gl3wInit();
 #include <pthread.h>
 
 #ifdef GPUCA_O2_LIB
+#if __has_include("../src/imgui.h")
```

Comment:
I would do it the other way around, otherwise you will not be able to test with the external DebugGUI until we remove the internal one.